High/KittyHighMD.cs

```diff
@@ -24,6 +24,12 @@ class KittyHighMD : KittyPL {
         public KittyHighMD() {
             Langs["md"] = this;
             Language = "MarkDown";
+            stringstart = "`";
+            stringend = "`";
+            KeyWords.Add("#");
+            KeyWords.Add("=");
+            KeyWords.Add("-");
```

Comment:
As single characters, they can be falsely highlighted in a text

VSCode starts highlighting the '=' when there are at least 3 consectutive '='

Marking only double characters would prevent this as normal occurrences of a '==' or '===' would be inside a code-block